solr/test-framework/src/java/org/apache/solr/SolrJettyTestBase.java

```diff
@@ -136,7 +136,7 @@ public synchronized SolrClient getSolrClient() {
    * options.
    */
   public SolrClient createNewSolrClient() {
```

Comment:
can we just delete this method?  Do we still need createNewSolrClient?

Reply:
Deleting it makes tests to fail with the error `java.lang.IllegalStateException: Connection pool shut down` I'm trying to crack the cause